example/full/table.properties

```diff
@@ -76,9 +76,9 @@ sleeper.table.compaction.strategy.class=sleeper.compaction.strategy.impl.SizeRat
 
 # The number of files to read in a compaction job. Note that the state store must support atomic
 # updates for this many files.
-# The DynamoDBStateStore must be able to atomically apply 2 updates to create the output files for a
-# splitting compaction, and 2 updates for each input file to mark them as ready for garbage
-# collection. There's a limit of 100 atomic updates, which equates to 48 files in a compaction.
+# The DynamoDBStateStore must be able to atomically apply 2 updates for each input file to mark them
+# as ready for garbage collection. There's a limit of 100 atomic updates, which equates to 48 files in
```

Comment:
Is this now 49 rather than 48? Each file needs the FileReference removed and the FileReferenceCount decremented, and a FileReference and a FileReferenceCount added for the output file. That means 49 input files is ok. It used to be 48 because a splitting compaction needed 2 output files.

Reply:
That makes sense reading the comment again, I've updated the comments and the validation for these